doc/source/GettingStarted/ConfiguringGafferForThirdPartyTools/index.md

```diff
@@ -55,17 +50,21 @@ To create the `ARNOLD_ROOT` environment variable in OSX:
 
 2. Add the line `export ARNOLD_ROOT=!ARNOLD_PATH_OSX!` and save.
 
-3. In a terminal (Finder > Go > Utilities > Terminal), reload the user configuration file, and then verify the variable is set:
+3. Open a terminal (Finder > Go > Utilities > Terminal).
+
+4. Test that the variable is set:
     
     ```bash
-    MacBook:~ user$ source ~/.bash_profile
     MacBook:~ user$ echo $ARNOLD_ROOT
-    !ARNOLD_PATH_OSX!
+    # !ARNOLD_PATH_OSX!
     ```
 
-4. The next time you start Gaffer, the Arnold nodes will be available from the node creation menu.
-    
-    <!-- TODO: ![Arnold node menu](images/arnoldNodes.png) -->
+
+### Verifying Arnold is loaded ###
+
+The next time you start Gaffer, the Arnold nodes will be available from the node creation menu (right-click inside the _Graph Editor_).
+
+<!-- TODO: ![Arnold node menu](images/arnoldNodes.png) -->
```

Comment:
@andrewkaufman How should we handle the 3rd party screenshots? Add a screengrab.py to gafferResources, then have them copied over?